frontends/web/src/routes/account/send/components/inputs/fiat-input.tsx

```diff
@@ -14,33 +14,38 @@
  * limitations under the License.
  */
 
+import { ChangeEvent } from 'react';
 import { useTranslation } from 'react-i18next';
 import { ConversionUnit } from '../../../../../api/account';
-import { Input } from '../../../../../components/forms';
-import { ChangeEvent } from 'react';
+import { NumberInput } from '../../../../../components/forms';
 
 type TProps = {
-    label: ConversionUnit;
-    onFiatChange: (event: ChangeEvent<HTMLInputElement>) => void;
-    disabled: boolean;
-    error?: string;
-    fiatAmount: string;
+  label: ConversionUnit;
+  onFiatChange: (amount: string) => void;
+  disabled: boolean;
+  error?: string;
+  fiatAmount: string;
 }
 
-export const FiatInput = ({ label, onFiatChange, disabled, error, fiatAmount }: TProps) => {
+export const FiatInput = ({
+  label,
+  onFiatChange,
+  disabled,
+  error,
+  fiatAmount,
+}: TProps) => {
   const { t } = useTranslation();
   return (
-    <Input
-      type="number"
+    <NumberInput
       step="any"
       min="0"
       label={label}
       id="fiatAmount"
-      onInput={onFiatChange}
+      onChange={(event: ChangeEvent<HTMLInputElement>) => onFiatChange(event.target.value)}
```

Comment:
I forgot if you recommended to use onChange or onInput  only from now on.. 

https://github.com/digitalbitbox/bitbox-wallet-app/commit/7e78cc7d8bb1bfb268452534d5a2367f65de164b